src/main/java/com/gizasystems/automationexercise/apis/ApisProducts.java

```diff
@@ -29,17 +38,27 @@ public ApisProducts searchProductApi(String searchProductFiled, String searchedP
                 .setParameters(formData, RestActions.ParametersType.FORM)
                 .setContentType(ContentType.URLENC)
                 .setTargetStatusCode(Apis.SUCCESS)
-                .perform();
+                .perform();    
         return this;
     }
 
     //////////////////// Validations \\\\\\\\\\\\\\\\\\\\
+  
+    @Step("Validate All products are listed")
+    public ApisProducts validateAllProductsAreListed(String productName) {
```

Comment:
I think this method name is misleading now after the changes as you are validating on only 1 product name now.

Reply:
done